spm_funcs.py

```diff
@@ -54,8 +54,14 @@ def get_spm_globals(fname):
     spm_vals : array
         SPM global metric for each 3D volume in the 4D image.
     """
-    # +++your code here+++
-    # return
+    # load the image by fname
+    img = nib.load(fname)
+    # get the data
+    data = img.get_fdata()
+    # compute spm global value for each volume
+    spm_vals = [spm_global(data[:, :, :, i]) for i in range(data.shape[-1])]
+    # return the result
+    return spm_vals
```

Comment:
What is the type of `spm_vals`?

If you read the docstring (line 54 above) it says `array`, which is very unclear but seems to refer to a `numpy.array`, however, you return a basic `list` here. Consider converting it to array before returning or else downstream methods could fail. For instance, with the current implementation you would not be able to do:
```Python
average_spm = get_spm_globals(nifti_file).mean()
```

Reply:
That makes sense. Thanks for the feedback! :+1: